spec/forms/search_developer_form_spec.rb

```diff
@@ -0,0 +1,84 @@
+
+require 'rails_helper'
+
+RSpec.describe SearchDeveloperForm do
+  describe '#search' do
+    before do
+      @language1 = create(:language, code: 'vn')
+      @language2 = create(:language, code: 'en')
+      @language3 = create(:language, code: 'jp')
+
+      @programming_language1 = create(:programming_language, name: 'ruby')
+      @programming_language2 = create(:programming_language, name: 'js')
+      @programming_language3 = create(:programming_language, name: 'c++')
+
+      @developer1 = create(:developer,
```

Comment:
@thupv 

`@developer1`, `@developer2`, `@developer3`
I think you should remove 

<img width="304" alt="screen shot 2018-04-04 at 14 50 16" src="https://user-images.githubusercontent.com/4110469/38295231-8692829e-3817-11e8-8662-aeae1106aca8.png">